app.js

```diff
@@ -7,11 +7,12 @@ const session = require('express-session');
 var indexRouter = require('./routes/index');
 var usersRouter = require('./routes/users');
 var auth = require('./routes/auth');
+var charactersRouter = require('./routes/characters')
 
 const app = express();
 
 mongoose.Promise = global.Promise;
-mongoose.connect('mongodb://localhost/test', { useNewUrlParser: true, useUnifiedTopology: true, useFindAndModify: false })
+mongoose.connect('mongodb+srv://EmilSchier:Minkodehertiler1%21@node-rest-shop-u8mvl.mongodb.net/test?retryWrites=true&w=majority', { useNewUrlParser: true, useUnifiedTopology: true, useFindAndModify: false })
```

Comment:
We need to put these environment variables in a separate file, that is ignored by git. That way we can each have have the secrets ;)

Reply:
I must admit i do nok quite know how to do this

Reply:
It's okay, I'll make some changes in another branch :)